posthog/models/filters/mixins/paths.py

```diff
@@ -75,6 +78,49 @@ def target_event(self) -> Tuple[Optional[PathType], Dict[str, str]]:
             return cast(PathType, PAGEVIEW_EVENT), {"event": PAGEVIEW_EVENT}
 
 
+class TargetEventsMixin(BaseParamMixin):
+    @cached_property
+    def target_events(self) -> List[str]:
+        return self._data.get(PATHS_INCLUDE_EVENT_TYPES, [])
```

Comment:
@liyiy - do you think there's a better way to pass information down from the frontend?

Right now it looks something like: https://github.com/PostHog/posthog/pull/5741/files#diff-9b53c3a3f635f0fa8c15a295b375b72af36bd9f60f638e833ca20acc609b7a57R325